transport_nantes/dashboard/views.py

```diff
@@ -27,3 +34,63 @@ class SignatureView(LoginRequiredMixin, FormView):
     """Form to generate a mail signature"""
     template_name = 'dashboard/signature.html'
     form_class = SignatureForm
+
+
+class EmailCampaignsDashboardView(PermissionRequiredMixin,
+                                  LoginRequiredMixin,
+                                  ListView):
+    """Dashboard of email campaigns"""
+    # Login & Permission
+    login_url = reverse_lazy("authentication:login")
+    permission_required = ('topicblog.tbe.may_send', 'topicblog.tbp.may_send')
```

Comment:
Doesn't this mean the user must have both permissions?  But surely having only one could permit being able to view the send records for that one type?